checker/raw_result.go

```diff
@@ -136,11 +136,28 @@ type VulnerabilitiesData struct {
 	Vulnerabilities []clients.Vulnerability
 }
 
+type SecurityPolicyInformationType string
+
+const (
+	// forms of security policy hints being evaluated.
+	SecurityPolicyInformationTypeEmail SecurityPolicyInformationType = "emailAddress"
+	SecurityPolicyInformationTypeLink  SecurityPolicyInformationType = "httpLink"
+	SecurityPolicyInformationTypeText  SecurityPolicyInformationType = "vuln & disclosure text"
```

Comment:
keep this camelcase, without spaces. This will ultimately be used in a field "name".

Reply:
changed `"vuln & disclosure text"` to `"vulnDisclosureText"`